api/src/services/streams-service.ts

```diff
@@ -199,4 +212,25 @@ export class StreamsService {
 		}
 		return seed;
 	}
+
+	toBytes(str: string): Uint8Array {
+		const bytes = new Uint8Array(str.length);
+		for (let i = 0; i < str.length; ++i) {
+			bytes[i] = str.charCodeAt(i);
+		}
+		return bytes;
+	}
+
+	fromBytes(bytes: any): string {
```

Comment:
These 2 functions look like a common helper functions, and can be moved into `src/utils`